src/App/App.js

```diff
@@ -17,7 +17,11 @@ const App = (props) => {
           <Route exact path='/packing-quiz' component={Quiz}/>
           <Route exact path='/packing-list' component={PackingList}/>
           <Route exact path='/saved-packing-lists' component={SavedView}/>
-          <Route path='/' component={Error} />
+          <Route path='/' render={() => {
+            return (<Error 
+              errorMessage='Oops! The page you are looking for does not exist.'
```

Comment:
We want this to eventually be dynamic right? We are passing a prop but then it's just a hardcoded string..

Reply:
It is dynamic! Its a different string depending on where the Error component is rendered.